Classes/ToxWrapper/OCTTox.h

```diff
@@ -389,6 +389,51 @@ extern const NSUInteger kOCTToxPublicKeyLength;
  */
 - (int)fileSendRequestWithFriendNumber:(int32_t)friendNumber fileName:(NSString *)fileName fileSize:(uint64_t)fileSize;
 
+/**
+ * Send a file control request
+ *
+ * @param friendNumber Friend number to send/receive file
+ * @param sendOrReceive is 0 if we want the control packet to target a file we are currently sending, 1 if it targets a file we are
+ * currently receiving.
+ * @param fileNumber Number of file to be sent/received
+ * @param controlType type of file control
+ * @param pointer on data
```

Comment:
Appledoc has following syntax `@param [param] [Description]`

So this should be changed to `@param data Pointer on data`